manticore/__main__.py

```diff
@@ -79,51 +78,38 @@ def parse_arguments():
 def main():
     args = parse_arguments()
 
-    m = Manticore(args.argv[0], args.argv[1:], disasm=args.disasm)
-    m.policy = args.policy
-    m.args = args
+    log.set_verbosity(args.v)
```

Comment:
I think we should still opt to use m.verbosity here. `log` isn't meant to be a public API :/

Reply:
Loggers are global singleton, manticore is an object of which you can have a bunch. m.set_verbosity is confusing. It'll let you think you control the logging of m while you are controlling a global thing that affects loggers for all the manticores alive. Unless we use a static method? Manticore.set_verbosity ?